bin/cyhy-simple

```diff
@@ -96,10 +97,10 @@ def get_location_details(db, gnis_id):
         sys.exit(-1)
 
 
-def fill_in(db, filename, force=False):
+def fill_in(db, file_pointer, force=False):
     request = db.RequestDoc()
     config = SafeConfigParser(allow_no_value=True)
-    config.read([filename])
+    config.readfp(file_pointer)
     # import IPython; IPython.embed() #<<< BREAKPOINT >>>
```

Comment:
As long as you're in here, can you please remove this unneeded line?
```suggestion
```

Reply:
There are a lot of those sprinkled in the repo:

```console
$ grep -ri ipython\; *
bin/cyhy-archive:    # import IPython; IPython.embed() #<<< BREAKPOINT >>>
bin/cyhy-mongo:    # import IPython; IPython.embed() #<<< BREAKPOINT >>>
bin/cyhy-simple:    # import IPython; IPython.embed() #<<< BREAKPOINT >>>
bin/cyhy-simple:    # import IPython; IPython.embed() #<<< BREAKPOINT >>>
bin/cyhy-ticket:    # import IPython; IPython.embed() #<<< BREAKPOINT >>>
cyhy/util/util.py:        # import IPython; IPython.embed() #<<< BREAKPOINT >>>
cyhy/test/test_host_state_manager.py:    # import IPython; IPython.embed() #<<< BREAKPOINT >>>
cyhy/test/test_nmap_handler.py:        # import IPython; IPython.embed() #<<< BREAKPOINT >>>
cyhy/test/test_ticket_manager.py:    # import IPython; IPython.embed() #<<< BREAKPOINT >>>
cyhy/core/yaml_config.py:# import IPython; IPython.embed() #<<< BREAKPOINT >>>
var/GNIS_data_import.py:    # import IPython; IPython.embed() #<<< BREAKPOINT >>>
```

Do you want me to remove all of the debugging breakpoints?

Reply:
Mind if I punt this into a more general cleanup PR? I'd like to run `isort` against this project because I detest how unhelpfully organized the the imports are and I was going to do a pass with `black` to make sure that was still up-to-date.

Reply:
Thats fine too.  Can you please create an issue in case this gets pushed aside by more urgent tasks?

Reply:
Please see https://github.com/cisagov/cyhy-core/issues/79.

Reply:
Thanks @mcdonnnj - resolving this conversation.